resources/definitions/Ender3_MicroSwiss_DirectDrive.def.json

```diff
@@ -0,0 +1,118 @@
+{
+    "name": "Ender-3 MicroSwiss DirectDrive",
+    "version": 2,
+    "inherits": "creality_ender3",
+    "metadata": {
+        "quality_definition": "creality_base",
+        "visible": true,
+        "platform": "creality_ender3.3mf",
+		"author": "GitHub.com/RFBomb",
```

Comment:
We indent with spaces, not tabs. You indent with a mix of them, it seems.

Reply:
I used VS-Code to make these changes, and am personally used to using tabs in my other work. I must've had the setting incorrect. I'll fix these and re-commit